kore/src/Kore/Builtin/Builtin.hs

```diff
@@ -17,6 +17,7 @@ builtin modules.
 module Kore.Builtin.Builtin
     ( Function
     , FunctionImplementation
+    , verifiers
```

Comment:
This is not an appropriate place to define these verifiers. Please make a new module.